x/observer/types/chain_params.go

```diff
@@ -46,109 +61,90 @@ func (cpl ChainParamsList) Validate() error {
-	}
-	if params.ConfirmationParams == nil {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "confirmation params cannot be nil")
+	if cp.ConfirmationCount == 0 {
+		return errors.New("ConfirmationCount must be greater than 0")
 	}
-	if params.ConfirmationParams.SafeInboundCount == 0 {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "SafeInboundCount must be greater than 0")
-	}
-	if params.ConfirmationParams.FastInboundCount > params.ConfirmationParams.SafeInboundCount {
-		return errorsmod.Wrapf(
-			sdkerrors.ErrInvalidRequest,
-			"FastInboundCount must be less than or equal to SafeInboundCount",
-		)
+	if cp.ConfirmationParams == nil {
+		return errors.New("confirmation params cannot be nil")
 	}
-	if params.ConfirmationParams.SafeOutboundCount == 0 {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "SafeOutboundCount must be greater than 0")
-	}
-	if params.ConfirmationParams.FastOutboundCount > params.ConfirmationParams.SafeOutboundCount {
-		return errorsmod.Wrapf(
-			sdkerrors.ErrInvalidRequest,
-			"FastOutboundCount must be less than or equal to SafeOutboundCount",
-		)
+	if err := cp.ConfirmationParams.Validate(); err != nil {
+		return errors.Wrap(err, "invalid confirmation params")
 	}
 
 	// validate tickers and intervals
-	if params.GasPriceTicker <= 0 || params.GasPriceTicker > 300 {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "GasPriceTicker %d out of range", params.GasPriceTicker)
+	if cp.GasPriceTicker <= 0 || cp.GasPriceTicker > 300 {
+		return fmt.Errorf("GasPriceTicker %d out of range", cp.GasPriceTicker)
 	}
-	if params.InboundTicker <= 0 || params.InboundTicker > 300 {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "InboundTicker %d out of range", params.InboundTicker)
+	if cp.InboundTicker <= 0 || cp.InboundTicker > 300 {
+		return fmt.Errorf("InboundTicker %d out of range", cp.InboundTicker)
 	}
-	if params.OutboundTicker <= 0 || params.OutboundTicker > 300 {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "OutboundTicker %d out of range", params.OutboundTicker)
+	if cp.OutboundTicker <= 0 || cp.OutboundTicker > 300 {
+		return fmt.Errorf("OutboundTicker %d out of range", cp.OutboundTicker)
 	}
-	if params.OutboundScheduleInterval == 0 || params.OutboundScheduleInterval > 100 { // 600 secs
-		return errorsmod.Wrapf(
-			sdkerrors.ErrInvalidRequest,
+	if cp.OutboundScheduleInterval == 0 || cp.OutboundScheduleInterval > 100 { // 600 secs
+		return fmt.Errorf(
+
 			"OutboundScheduleInterval %d out of range",
-			params.OutboundScheduleInterval,
+			cp.OutboundScheduleInterval,
 		)
 	}
-	if params.OutboundScheduleLookahead == 0 || params.OutboundScheduleLookahead > 500 { // 500 cctxs
-		return errorsmod.Wrapf(
-			sdkerrors.ErrInvalidRequest,
+	if cp.OutboundScheduleLookahead == 0 || cp.OutboundScheduleLookahead > 500 { // 500 cctxs
+		return fmt.Errorf(
+
 			"OutboundScheduleLookahead %d out of range",
-			params.OutboundScheduleLookahead,
+			cp.OutboundScheduleLookahead,
```

Comment:
I feel like we can remove the upper limits for these values, they are set by a policy after all. 
Removing might simply the validation logic. 
Does not need to be done in this pr though ,